src/v/cluster/topics_frontend.cc

```diff
@@ -1893,4 +1899,31 @@ topics_frontend::get_partition_state(model::ntp ntp) {
     co_return results;
 }
 
+void topics_frontend::print_rf_warning_message() {
+    // reduce log clutter and only run warning on shard 0
+    if (ss::this_shard_id() != 0) {
+        return;
+    }
```

Comment:
This should either:
1) Go in the docs for this method
2) Be lifted to the callers.